### PR TITLE
John conroy/fix error page html titles

### LINKED ISSUE #1213: Error pages are missing tite
Error page titles are all " | HuBMAP".

They should be "<Error Type> | HuBMAP", e.g., "Page Not Found | HuBMAP".

image

### PR DESCRIPTION
**Pull request by john-conroy**
Fix #1213. This should fix error page html titles for all flask registered error routes.

### CODE REVIEW

**Review by mccalluc — COMMENTED** [review]
Sorry to be picky, but could we add a second argument to `render_react_error`, and specify the messages where it's called from `main.py`, for instance:
```diff
def bad_request(e):
    '''A 400 means the request to the API failed.'''
-    return render_react_error(400)
+    return render_react_error(400, 'Bad Request')
```

We want to avoid having to keep two parallel lists in sync (the functions, and the dict).

**Comment by john-conroy** [comment]
Not picky, thanks for the suggestion. It was just one of those moments where you overlook the obvious solution.

File: context/app/main.py [hunk 1]
```diff
@@ -54,15 +56,15 @@ def create_app(testing=False):
     app.register_error_handler(404, not_found)
     app.register_error_handler(504, gateway_timeout)
 
-    @app.context_processor
+    @ app.context_processor
```
Comment (mccalluc) [inline context/app/main.py:59]:
```suggestion
    @app.context_processor
```
I think no space is more usual?

File: context/app/main.py [hunk 2]
```diff
@@ -54,15 +56,15 @@ def create_app(testing=False):
...
     def inject_template_globals():
         return {
             'is_authenticated': session.get('is_authenticated'),
             'nexus_token': session.get('nexus_token'),
             'user_email': session.get('user_email')
         }
 
-    @app.before_request
+    @ app.before_request
```
Comment (mccalluc) [inline context/app/main.py:67]:
```suggestion
    @app.before_request
```

**Review by mccalluc — COMMENTED** [review]
Thanks for the tweaks! Probably revert adding the spaces after `@` and then merge.

**Pull request merged**